web/blog/vast-v3.0/index.md

```diff
@@ -407,5 +407,5 @@ package is called `vast` and also available [separately on PyPI][vast-pypi].
 ### Extractor Predicates
 
 Extractors can now be used where predicates are expected to test for the
-existance of a field or type. For example, `x` and `:T` expand to `x != nil`
-and `:T != nil`, respectively.
+existance of a field or type. For example, `x` and `:T` expand to `x != null`
+and `:T != null`, respectively.
```

Comment:
Can you also adapt the changelog entries for #2776 and #2984 to use `null` over `nil`?

Reply:
I thought those weren't allowed to be changed anymore, with `rc1` being out. But I changed them in https://github.com/tenzir/vast/pull/2999/commits/c7a69888d211895c004793352d852b0526a292f8.

Reply:
We don't keep the -rc sections in the changelog around, they disappear when we cut the actual release. So that's fine given that an rc2 is imminent.